homework_weekend6/src/homework_weekend6/task1/Main.java

```diff
@@ -0,0 +1,87 @@
+package homework_weekend6.task1;
+
+import java.awt.Font;
+import java.awt.GridLayout;
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+
+import javax.swing.JButton;
+import javax.swing.JFrame;
+import javax.swing.JOptionPane;
+
+/**
+ * Creates GUI application that us a window with a row of buttons, 
+ * named with numbers from 0 to 9. User is allowed to pick up (by pressing buttons)
+ * three numbers. After that, to user will be presented another window with printed 
+ * out three digits number formed from selected numbers.
+ * @author ajla
+ *
+ */
+public class Main extends JFrame {
+
+	private static final long serialVersionUID = 993668091274336134L;
+	private static final int PICKED_NUMBERS_COUNTER = 1;
+	private static final int MAX_NUMBERS = 3;
+	private static final int NUMBER_OF_BUTTONS = 10;
+	private JButton[] buttons = new JButton[NUMBER_OF_BUTTONS];
+	private String res = "";
```

Comment:
Variable name should be more descriptive. 
Maybe use StringBuilder to avoid string concatenation.